File: JavaScript/2016/6.js
    const _ = require('lodash');
    const lines = require('../getInput')(6, 2016).trim().split('\n');

    const getResult = funcName =>
      _(_.range(8)).map(i => _(lines).map(i).countBy().toPairs()[funcName](1)).map(0).join('');

    console.log(['maxBy', 'minBy'].map(getResult));

File: Python/2016/6.py
from collections import Counter

with open('2016/6.txt') as f:
    lines = f.read().strip().split('\n')
    columns = [[line[i] for line in lines] for i in range(8)]

print(''.join([Counter(col).most_common()[0][0] for col in columns]))
print(''.join([Counter(col).most_common()[-1][0] for col in columns]))
